ui/tailwind.config.js

```diff
@@ -54,6 +54,18 @@ module.exports = {
                 800: 'hsl(331, 87%, 15%)', // #470525
                 900: 'hsl(333, 85%, 5%)' // #18020c
             },
+            green: {
```

Comment:
hsl versions preferred to match the other colors, though it doesn't effect the site

Reply:
Yeah, will sort that, i was being lazy